jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/OdePropertiesTest.java

```diff
@@ -108,10 +93,9 @@ public void testSettersAndGetters() {
             String testRsuUsername = "testRsuUsername";
             String testRsuPassword = "testRsuPassword";
 
+            testOdeProperties.setHostIP("test-host");
             testOdeProperties.setEnv(mockEnv);
             testOdeProperties.setEnvironment(mockEnv);
-            testOdeProperties.setKafkaBrokers(testKafkaBrokers);
-            testOdeProperties.setKafkaProducerType(testKafkaProducerType);
```

Comment:
note: now all moved to [OdeKafkaProprtiesTest](https://github.com/CDOT-CV/jpo-ode/pull/110/files#diff-b771b678401cde76ebe39b58c11c2ce155b9092e84fbfba935a4555cb609728b)